rero_ils/modules/documents/dojson/contrib/marc21tojson/slsp/model.py

```diff
@@ -389,9 +389,10 @@ def marc21_to_subjects_6XX(self, key, value):
             '610': False,
             '611': True
         }
-        if tag_key in ['610', '611']:
-            subject['conference'] = conference_per_tag[tag_key]
-        elif tag_key in ['600t', '610t', '611t']:
+        # if tag_key in ['610', '611']:
+        #     subject['conference'] = conference_per_tag[tag_key]
+        # elif tag_key in ['600t', '610t', '611t']:
```

Comment:
you could removed these lines

Reply:
Yes, but maybe you need it for local entities.